package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-edge-devtools",
-    "displayName": "Elements for Microsoft Edge (Chromium)",
-    "description": "Use the Microsoft Edge (Chromium) Elements tool from within VS Code to see your site's runtime HTML structure, alter its layout, and fix styling issues.",
+    "displayName": "Microsoft Edge (Chromium) Tools for VSCode",
+    "description": "Use the Microsoft Edge (Chromium) Tools from within VS Code to see your site's runtime HTML structure, alter its layout, fix styling issues as well as see your site's network requests.",
     "version": "1.0.7",

```

Comment:
"version": "1.0.8"

Reply:
Good catch.  Changed, updated package-lock.json and checking in